docs/index.asciidoc

```diff
@@ -59,11 +58,8 @@ database by default.
 ==== Database Auto-update
 
 This plugin bundles Creative Commons (CC) license databases.
-Logstash checks for database updates every day. It downloads the latest and can replace the old database
+By default, Logstash checks for database updates every day and upgrades from the CC database licence to the EULA version automatically. It downloads the latest version and replaces old databases
```

Comment:
Has the behavior changed from the previous version of the docs?  

This wording is confusing because it's mixing a daily update with a one time conversion to the EULA license.

Reply:
Hum, that's true, this behavior didn't actually change from the previous version. Considering we have now an option to disable the auto-update, I tried to explain here that the license also auto-upgrades from CC to EULA when the download is successfully completed. 

What about changing it to something like: 
```
When the auto-update feature is enabled, Logstash checks for database updates every day. It downloads the latest and can replace the old database
```

In that case, I could also restore the removed NOTE to something like:
```
NOTE: If the auto-update feature is disabled or the database has never been updated successfully, as in air-gapped environments, Logstash can use CC license databases indefinitely.
```

What do you think?

Reply:
I like where you're heading with this.  If I'm reading your proposed changes correctly, we still need to account for the license conversion.  Something like: 

`If the auto updates feature is enabled, Logstash upgrades from the CC database license to the EULA version on the first download. `